client/src/components/Header.js

```diff
@@ -13,6 +16,67 @@ const Header = () => {
       </nav>
     </div>
   );
+
 };
 
+class Header extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      auth: '',
+    };
+  }
+
+  componentDidMount() {
+    axios
+      .get('auth/current_user')
+      .then(user => {
+        this.setState({ auth: user.data });
+        console.log(user);
+      })
+      .catch(err => {
+        console.log(err);
+      });
+  }
+
+  renderContent() {
+    switch (this.state.auth) {
+      case null:
+        return;
```

Comment:
again should return some sort of error, or something saying to login again

Reply:
The null only occurs within the split second when React is updating the state, at that time, the bar shows nothing instead of displaying an error. 
I have taken care of not being able to login in the lower cases.